DOCUMENTATION.md

```diff
@@ -1902,6 +1902,12 @@ added and the head is where the items are overwritten or expired. You can reach
 which is mapped to the elements between the head and tail (inclusive) of the Ringbuffer. For details, see the
 [Ringbuffer section](https://docs.hazelcast.com/hazelcast/latest/data-structures/ringbuffer) in the Hazelcast Reference Manual.
 
+`filter` argument is provided for to only select items that need to be read. If
```

Comment:
I don't remember if I said to add some documentation here but I think this type of documentation will be better in the API documentation.

Reply:
You can add a short usage here or to the code samples and give a link to it.



something like:

You can use `Ringbuffer.readMany` function to read more than one item. You can use the `filter` method to filter out some items if you want. The filter class have to be in the class path of members. For an example usage check out this(add a link to `this`) code sample.